### PR TITLE
Check target.checked is undefined (fix #41)

### LINKED ISSUE #41: AntD switch is not correctly if type is checkbox.
**Describe the bug**
When I use Antd switch with RHFInput, I think it is right to set type to `checkbox`. then, Switch is not working. but if type is `input`, it is working.

**To Reproduce**
Checkout below codesandbox.

**Codesandbox link**
https://codesandbox.io/s/rhf-input-with-antd-switch-2rdwx

**Expected behavior**
It should be work with `type="checkbox"` not `type="input"`

**Screenshots**

**Desktop (please complete the following information):**
 - Chrome 78 on catalina

**Smartphone (please complete the following information):**

**Additional context**
I think it is a bug. so I will create a PR for resolve this issue. can I?

### PR DESCRIPTION
**Pull request by iamchanii**
Fix: #41

### CODE REVIEW

**Comment by bluebill1049** [comment]
i will publish a beta now :)

**Comment by bluebill1049** [comment]
"version": "1.1.10.beta.1", let me know if this fix the issue 👍

File: src/index.tsx [hunk 1]
```diff
@@ -12,7 +12,9 @@ function getValue(target: any, { isCheckbox }: { isCheckbox: boolean }) {
 
   return target
     ? isCheckbox
-      ? target.checked
+      ? isUndefined(target.checked)
+        ? target.checked
```
Comment (iamchanii) [inline src/index.tsx:16]: @bluebill1049 Omg, I missed a point. It should be `target` not `target.checked` ... Sorry! I will create new PR...

Comment (bluebill1049) [reply src/index.tsx:16]: oops no worries! i will release another beta once it's merged in

**Pull request merged**